multiwfn/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, makeWrapper, requireFile, autoPatchelfHook, unzip, writeScriptBin, fetchurl
```

Comment:
regarding formatting: both versions, single or multiple packages per line, are OK. However, I would prefer to stick to one version.